src/test/java/LoginTests.java

```diff
@@ -1,57 +1,44 @@
 import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
 import org.testng.Assert;
-import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
-import pages.HomePage;
-import pages.LoginPage;
 
-import java.util.List;
-
-
-public class LoginTests extends BaseTest {
-
-
-    @Test(dataProvider = "IncorrectLoginProviders")
-    public void negativeLoginTests(String email, String password) {
-        LoginPage loginPage = new LoginPage(getThreadLocal());
-        loginPage.enterEmail(email);
-        loginPage.enterPassword(password);
-        loginPage.clickLoginBtn();
-        Assert.assertEquals(driver.getCurrentUrl(), url);
-    }
+import java.time.Duration;
 
+public class LoginTests  {
     @Test
-    public void loginSucceedTest() throws InterruptedException {
-        LoginPage loginPage = new LoginPage(getThreadLocal());
-        HomePage homePage = new HomePage(getThreadLocal());
-        loginPage.enterEmail("demo@class.com");
-        loginPage.enterPassword("te$t$tudent");
-        loginPage.clickLoginBtn();
-        Thread.sleep(3000);
-        Assert.assertTrue(homePage.getAvatar());
-    }
+    public static void loginSucceedTest()throws  InterruptedException {
+
+//      Added ChromeOptions argument below to fix websocket error
+
```

Comment:
extending the BaseTest would be enough to initialize the instance of the browser, you do not need to add chrome options and a new chrome driver here.